submarine-server/server-api/src/main/java/org/apache/submarine/server/api/exception/SubmitterException.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.submarine.server.api.exception;
+
+import org.apache.submarine.commons.utils.exception.SubmarineRuntimeException;
+
+/**
+ * SubmitterException is caused by internal or upstream server running error.
+ */
+public class SubmitterException extends SubmarineRuntimeException {
+  private final int code;
```

Comment:
Now the extended `*Exception` makes the system very complicated.
It is sufficient to use `SubmarineRuntimeException` in the commons-utils module.